libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -25,10 +26,12 @@
     <div>
       <button
         mat-icon-button
-        data-testing="toggle-finish-book-cta"
-        [ngClass]="b.finished ? 'mat-icon-button-green' : 'mat-icon-button-gray'"
+        data-testing="finish-book-cta"
+        [ngClass]="'mat-icon-button-green'"
         [attr.aria-label]="'Mark Read ' + b.title + ' from reading list'"
```

Comment:
Please change it to : `[attr.aria-label]="'Mark ' + b.title + ' book as finished in reading list'"`